universal-login-react/src/ui/Transfer/Amount/TransferAmount.tsx

```diff
@@ -1,75 +1,52 @@
-import './../../styles/transferAmountDefaults.sass';
-import {getStyleForTopLevelComponent} from '../../../core/utils/getStyleForTopLevelComponent';
-import {useAsyncEffect} from '../../hooks/useAsyncEffect';
+import {TransferDetails, TokenDetailsWithBalance, TokenDetails} from '@universal-login/commons';
 import {FeatureFlag} from '../../commons/FeatureFlag';
 
 export interface TransferAmountProps {
   deployedWallet: DeployedWallet;
-  onSelectRecipientClick: () => void;
+  tokenDetailsWithBalance: TokenDetailsWithBalance[];
   updateTransferDetailsWith: (transferDetails: Partial<TransferDetails>) => void;
   tokenDetails: TokenDetails;
+  amountError: boolean;
+  setAmountError: (isAmountInvalid: boolean) => void;
   transferAmountClassName?: string;
 }
 
-export const TransferAmount = ({deployedWallet, onSelectRecipientClick, updateTransferDetailsWith, tokenDetails, transferAmountClassName}: TransferAmountProps) => {
-  const [tokenDetailsWithBalance, setTokenDetailsWithBalance] = useState<TokenDetailsWithBalance[]>([]);
-  const [validAmount, setValidAmount] = useState(true);
-  const [enableButton, setEnableButton] = useState(false);
-  const {sdk, contractAddress} = deployedWallet;
-
-  useAsyncEffect(() => sdk.subscribeToBalances(contractAddress, setTokenDetailsWithBalance), []);
-  const balance = getBalanceOf(tokenDetails.symbol, tokenDetailsWithBalance);
-
-  const validateAndUpdateTransferDetails = (amount: string) => {
-    if (balance && isValidAmount(amount, balance)) {
-      setValidAmount(true);
-      setEnableButton(true);
-      updateTransferDetailsWith({amount});
-    } else {
-      setValidAmount(false);
-      setEnableButton(false);
-    }
+export const TransferAmount = ({deployedWallet, tokenDetailsWithBalance, updateTransferDetailsWith, tokenDetails, amountError, setAmountError, transferAmountClassName}: TransferAmountProps) => {
+  const onChange = (amount: string) => {
+    setAmountError(false);
+    updateTransferDetailsWith({amount});
   };
 
   return (
-    <div className="universal-login-amount">
-      <div className={getStyleForTopLevelComponent(transferAmountClassName)}>
-        <div className="transfer-amount">
-          <TransferDropdown
-            sdk={sdk}
-            tokenDetailsWithBalance={tokenDetailsWithBalance}
-            tokenDetails={tokenDetails}
-            setToken={(token: TokenDetails) => updateTransferDetailsWith({transferToken: token.address})}
-            className={transferAmountClassName}
+    <>
+      <TransferDropdown
+        sdk={deployedWallet.sdk}
+        tokenDetailsWithBalance={tokenDetailsWithBalance}
+        tokenDetails={tokenDetails}
+        setToken={(token: TokenDetails) => updateTransferDetailsWith({transferToken: token.address})}
+        className={transferAmountClassName}
+      />
+      <div className="transfer-amount-row">
+        <label className="transfer-amount-label" htmlFor="amount-eth">How much are you sending?</label>
```

Comment:
`<label>Amount</label>` will be constistent with 'Recipienist'

Reply:
👍 Done